index.html

```diff
@@ -8,32 +8,183 @@
+                    </ul>
+                    <button type="button">
+                        See Project
+                    </button>
+                </div>
+                <div class="card">
+                    <img class="preview" src="./images/SnapshootPortfolio4.png" alt="Website Preview">
+                    <h2 >
+                        Multi-Post Stories
+                    </h2>
+                    <div class="type_and_date">
+                        <span>CANOPY</span>
+                        <span>●</span>
+                        <span>Front End Dev</span>
+                        <span>●</span>
+                        <span>2023</span>
+                    </div>
+                    <p>
+                        A daily selection of privately 
+                        personalized reads; no accounts or
+                        sign-ups required.
+                    </p>
+                    <ul class="prog_lang">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button type="button">
+                        See Project
+                    </button>
+                </div>
+            </section>
+                     
+                <section class="aboutme">
+                    <h2>About Myself</h2>
+                    <p> 
+                        Hello I'm a software developer! I can help you 
+                        build a product, feature or website Look 
+                        through some of my work and 
+                        experience! If you like what you see and
+                        have a project you need coded, don't 
+                        hesitate to contact me. 
+                    </p>
+                    <h3>
+                        <a class="connect_link" href="">LET'S CONNECT</a>
+                    </h3>
+                    <ul class="icons">
+                        <li><a href="https://twitter.com/MohametAlmeari"><i class="fa fa-twitter"></i></a></li>
+                        <li><a href="https://www.linkedin.com/in/mohamet-almeari"><i class="fa fa-linkedin"></i></a></li>
+                        <li><a href="https://github.com/mohametalmeari"><i class="fa fa-github"></i></a></li>
+                        <li><a href=""><i class="fa fa-instagram"></i></a></li>
+                    </ul>
+
+                    <button type="button">
+                        Get my resume
+                    </button>
+                    <ul class="footer_list">
+                        <li>
+                            Languages <i class="fa fa-angle-right"></i>
+                        </li>
+                        <li>
+                            Frameworks <i class="fa fa-angle-right"></i>
+                        </li>
+                        <li>
+                            Skills <i class="fa fa-angle-right"></i>
+                        </li>
```

Comment:
[OPTIONAL] Kindly add the images of the skill section as requested by the previous reviewer since it is part of the project requirement and it will make your project more complete. 🙏